main/res/values/strings_not_translatable.xml

```diff
@@ -85,7 +85,8 @@
 	</string>
 
     <!-- cache details -->
-    <string name="favorite_count" translatable="false">%1$d×</string>
+    <string name="favorite_count" translatable="false">%1$d</string>
```

Comment:
> Probably something like this. Also removed "x" from the non-percent recourse since it's now only for zero.

The resource is also used in other places where the cache count is greater than 0.

Reply:
I don't like this weird "x"